src/ServiceManagement/Services/Commands.Utilities/Azure.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '5.0.0'
+ModuleVersion = '5.1.0'
```

Comment:
@markcowl should this PR bump the version of `Azure` or should we bump it externally with our tooling?

Reply:
We should bump it.